source/libtomo/accel/common.hh

```diff
@@ -92,19 +93,19 @@ CreateThreadPool(unique_thread_pool_t& tp, num_threads_t& pool_size)
     auto tid = GetThisThreadID();
 
     // initialize the thread-local data information
-    auto& thread_data = ThreadData::GetInstance();
+    auto& thread_data = PTL::ThreadData::GetInstance();
     if(!thread_data)
-        thread_data.reset(new ThreadData(tp.get()));
+        thread_data = new PTL::ThreadData(tp.get());
```

Comment:
This was the only API transition that I was unsure about. Previously, GetInstance returned a std::unique_ptr, but now it just returns a regular pointer. I think assignment is the correct operation here? @jrmadsen